feat: Add center cards to theme pages

vgeorge opened the pull request:
Contributes to https://github.com/NASA-IMPACT/veda-ui/issues/1733.

Changes:

- Introduces `<CenterCardBlock>` component, that can be used in mdx pages (it accepts center ids to keep single source of truth in `app/constants/DATA_CENTERS`
- Style fixes to make card image and content to appear consistent in different pages and screen sizes
- Updated frontmatter and add center cards block to sea leven change and greenhouse gases theme page

This is ready for review.

dzole0311 commented:
lgtm, @vgeorge can we just add the links:

1. Greenhouse Gas Center card: https://earth.gov/ghgcenter
2. Global Sea Level Change Portal: https://earth.gov/sealevel
3. National Sea Level Rise center: https://earth.gov/sealevel/us/

vgeorge commented:
@dzole0311 thanks for the review! I added the links. I also removed the EIC card from the Dashboard. Am I right to assume the EIC is this website (earth.gov)? If so, I don't think we should display a card for the website itself. 

If this is ready to merge, I need a rubberstamp to proceed. 😅
